Misc/NEWS.d/next/Library/2018-07-16-11-35-00.bpo-34098.CVjiuZ.rst

```diff
@@ -0,0 +1,2 @@
+Embed stringification of remote traceback in local traceback when a method
```

Comment:
Please, use ``method`` when formatting classes and such. For example:

> executed via ``multiprocessing.BaseManager`` raises

Also, add a colon at the end of the line. :)

Reply:
@pablogsal done. used double backticks like I've found in other NEWS entries

Reply:
Please add "Patch by Guilherme Salgado" at the end. :)